app/scripts/components/viewer3dcomponent.js

```diff
@@ -82,12 +82,13 @@ var exports = (function () {
         "deep": true,
         "handler": function () {
           let suid = _.get(this.activeModel, "data.suid");
+          let progress = _.get(this.activeModel, "progress", 0);
           let timestep = _.get(this.activeModel, "data.info.delta_fringe_images.images", []).length;
           let sedimentClass = _.get(this.activeModel, "data.parameters.composition.value");
 
           if (suid !== this.curSuid && timestep !== 0 && sedimentClass !== undefined) {
             this.curSuid = suid;
-            this.curFrameLength = this.curTimeStep = timestep;
```

Comment:
I believe the problem is not in the final image but that `_.get(this.activeModel, "data.info.delta_fringe_images.images", []).length` is 1 based. The viewer expects 0 based. I would rewrite this to something like:
```
let nImages = _.get(this.activeModel, "data.info.delta_fringe_images.images", []).length; 
// Convert from 1 based to 0 based (default to 0 for empty images).
let timestep = nImages > 0  ? nImages - 1 : 0
// viewer expects a 0 based timestep (would you call this timestep or timeIndex)
```

And perhaps replace the `timestep !== 0` check with `nImages > 0`. 

The 0 based index is clear if you look at line 129 in SliceHandler.